src/components/Filter/FilterInput.vue

```diff
@@ -79,6 +79,7 @@
               @keydown.meta.a.prevent.stop="selectInputAndTags"
               @keydown.ctrl.a.prevent="selectInputAndTags"
               @keydown.exact="inputKeydownHandler"
+              @keydown.191.exact.stop="inputKeydownHandler"
```

Comment:
I am not sure the FilterInput should handle specially the `/` key, if I am getting this correct.

Reply:
The reason behind this change was to stop the propagation of the event and prevent the `onKeydown` from `DocumentationTopic` from getting called. Without this, the modal gets opened when the user tries to type `/` on the filter input. But since you mention it, there's probably a better way to solve this.

Reply:
Resolved [dccfec7](https://github.com/apple/swift-docc-render/pull/396/commits/dccfec7956dd3d7b98fb0138097aaa05f31681e7)